examples/playwright/src/tests/theia-notebook-editor.test.ts

```diff
@@ -0,0 +1,165 @@
+// *****************************************************************************
+// Copyright (C) 2024 TypeFox GmbH and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { PlaywrightWorkerArgs, expect, test } from '@playwright/test';
+import { TheiaApp } from '../theia-app';
+import { TheiaAppLoader, TheiaPlaywrightTestConfig } from '../theia-app-loader';
+import { TheiaNotebookCell } from '../theia-notebook-cell';
+import { TheiaNotebookEditor } from '../theia-notebook-editor';
+import { TheiaWorkspace } from '../theia-workspace';
+import path = require('path');
+
+test.describe('Theia Notebook Editor interaction', () => {
+
+    let app: TheiaApp;
+    let editor: TheiaNotebookEditor;
+
+    test.beforeAll(async ({ playwright, browser }) => {
+        app = await loadApp({ playwright, browser });
+    });
+
+    test.afterAll(async () => {
+        await app.page.close();
+    });
+
+    test.afterEach(async () => {
+        if (editor) {
+            await editor.closeWithoutSave();
+        }
+    });
+
+    test('kernels are installed', async () => {
+        editor = await app.openEditor('sample.ipynb', TheiaNotebookEditor);
+        await editor.tabLocator().click(); // force activate
```

Comment:
I think it might be better to move that into the `TheiaNotebookEditor` page object, if this is really necessary. Otherwise every test would have to be aware that this needs to be called before subsequent interactions.

Reply:
It was an attempt to fix flakiness on first run. I didn't found out why, but right after the start and opening an editor the focus jumps to the browser address field. Now I think it was because the "Workspace trust" message dialog was popping-up. It should no longer be necessary as the check is disabled in the example project package.json. I will remove this lines, let's see if the tests are passing in CI.